packages/sui-mock/README.md

```diff
@@ -0,0 +1,135 @@
+# sui-mock
+
+### Installation
+
+```sh
+npm install @s-ui/mock --save-dev
+```
+
+### Mockers
+
+#### **Request mocking with `setupMocker`**
+
+It returns all methods included in `setupWorker` and `setupServer`. It will work in browser and server sides.
+
+👉 Check `setupWorker` and `setupServer` in [MSW docs](https://mswjs.io/docs/api/).
+
+
+### Usage
+
+#### 1. Create Request handlers
+Create a `./mocks` folder in your project root and create a `index.js` file inside it.
```

Comment:
First of all we need to define the application request handlers. Request handler is a function that determines whether an outgoing request should be mocked, and specifies its mocked response.